Assets/Scripts/TES/MorrowindEngine.cs

```diff
@@ -854,7 +860,7 @@ private GameObject CreatePlayerCamera(Vector3 position)
 		{
 			var camera = GameObjectUtils.CreateMainCamera(position, Quaternion.identity);
 			camera.GetComponent<Camera>().cullingMask = ~(1 << markerLayer);
-
+			Camera.main.renderingPath = RenderingPath.DeferredShading;
```

Comment:
My computer is terrible, so I avoided using deferred shading. At the very least I'd like to add an option to enable/disable deferred rendering in the inspector, or better, in the main menu. So I might remove this line from the request later and add an option for deferred rendering in another commit.